src/domains/pay-wallet-app/05_aks_middleware_tools.tf

```
@@ -0,0 +1,53 @@
+module "tls_checker" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//tls_checker?ref=v6.3.0"
+
+  https_endpoint                         = local.wallet_hostname
```

```suggestion
  https_endpoint                         = local.payment_wallet_hostname
```

what about refactoring also those variable names?